### DISCUSSION TITLE
Vulnerability Reporting for MIPS Added to Kernel

### DISCUSSION THREAD

**Discussion by owlshrimp** [speed47/spectre-meltdown-checker#554] [Other vulnerabilities / information]
https://www.phoronix.com/scan.php?page=news_item&px=Linux-MIPS-Vulnerabilities

There is now support for reporting various spectre vulnerabilities on MIPS hardware. However, the state of most vulnerabilities on most hardware is "unknown" by default as most MIPS vendors are either unresponsive or dead.

**Comment by speed47** [comment]
Thanks for the insight, leaving open with the tag "information"
